sky/backends/cloud_vm_ray_backend.py

```diff
@@ -4495,6 +4495,7 @@ def set_autostop(self,
 
                     # down = False is the default, but warn the user in case
                     # they have explicitly specified it.
+                    # TODO(cooperc): Fix for new autostop stuff.
```

Comment:
do we need to fix this?

Reply:
Since it's just a warning, I don't think it's a blocker.